torch/_refs/__init__.py

```diff
@@ -4428,7 +4426,7 @@ def logspace(
         pin_memory=pin_memory,
         requires_grad=requires_grad,
     )
-    return prims.to_dtype(torch.pow(base, ret), dtype)
+    return _maybe_convert_to_dtype(torch.pow(base, ret), dtype)
```

Comment:
Even better, you can pass `dtype=dtype` to `linspace` and it should work as expected.

Reply:
I would presume the `pow` calculation is intentionally done in float64 for precision reasons.

Reply:
aaaand, I miss the pow. Sorry for that.